docs/source/serialization.rst

```diff
@@ -24,6 +24,10 @@ Python                                  JSON
 ``DateTime('2015/11/23 19:45:55')``     ``'2015-11-23T19:45:55'``
 ======================================= ======================================
 
+.. warning::
```

Comment:
The only thing to keep in mind by using this approach is when used in combination with zope.schema date validators. If that's the case, you should also use a datetime object that also contains offset-aware object as the validator value.